packages/labs/gen-wrapper-vue/src/lib/vite.config-template.ts

```diff
@@ -30,7 +30,7 @@ export default {
   build: {
     rollupOptions: {
       // Ensures no deps are bundled with this build.
-      external: () => true,
+      external: (id: string) => !!id.match(/^(vue|@lit.*|lit)$/),
```

Comment:
Seems like this should be any bare module specifier (or anything that's not relative...?) `!id.match(/^\./)`.

Reply:
Done with slight modification.